docs/azuredisk-issues.md

```diff
@@ -236,6 +239,7 @@ spec:
     runAsUser: 0
     fsGroup: 0
 ```
+ > Note: Since gid & uid is mounted as 0(root) by default, if set as non-root(e.g. 1000), k8s will use chown/chmod to change all dir/files under that disk, this is a time consuming job, whch would make mount device very slow, in this issue: [Timeout expired waiting for volumes to attach](https://github.com/kubernetes/kubernetes/issues/67014#issuecomment-413546283), it costs about 10 min for chown/chmod operation complete.
```

Comment:
whch -> which

Reply:
only chown? when does chmod applies?

Reply:
depends on what value gid or uid is set, k8s will use chown/chmod to apply.

Reply:
fixed